constants.js

```diff
@@ -111,7 +111,6 @@ const ASSERTION_STATES = {
 
 const CONTENT_TYPES = {
     PUBLIC: 'public',
-    PRIVATE: 'private',
```

Comment:
i wouldn't really remove it, there's a world where this can be useful in the future